src/pages/folderPage/FolderPage.js

```diff
@@ -1,16 +1,88 @@
-import React from "react";
+import React, { useState } from "react";
 import Nav from "./Nav";
 import Header from "./Header";
 import Article from "../../ui/Article";
 import Footer from "../../ui/Footer";
+import AddLink from "../../components/addLink/AddLink";
+import ModalBackground from "../../components/modal/ModalBackground";
+import ModalListButton from "../../components/modal/ModalListButton";
+import Modal from "../../components/modal/Modal";
 
 const FolderPage = () => {
+  const [data, setData] = useState();
+  const [isAddLinkClicked, setIsAddLinkClicked] = useState(false);
+  const [addLinkValue, setAddLinkValue] = useState("");
+  function getData(data) {
+    setData(data);
+  }
+  function handleAddLinkClick(e) {
+    e.preventDefault();
+    if (addLinkValue !== "") {
+      setIsAddLinkClicked(!isAddLinkClicked);
+    }
+  }
+  function handleModalListButton(e) {
+    e.preventDefault();
+  }
+
   return (
     <>
       <Nav />
-      <Header />
+      <header style={{ padding: "6rem 0 9rem 0" }}>
+        <AddLink
+          onClick={handleAddLinkClick}
+          setAddLinkValue={setAddLinkValue}
+        />
+      </header>
+      <Header getData={getData} />
       <Article />
       <Footer />
+      {isAddLinkClicked ? (
```

Comment:
이런건 삼항 연산자보다 요렇게 바꿔주시는게 더 간결하지않을까싶네요! ㅎㅎ
```suggestion
      {isAddLinkClicked && (
```